### PR TITLE
fix translations

### PR DESCRIPTION
**Pull request by maiko3tattun**
- Delete unused text
- Uniformity of indentation
- Reflects untranslated text in each language

This PR includes #730.
image

### CODE REVIEW

File: OpenUtau/Strings/Strings.axaml [hunk 1]
```diff
@@ -188,7 +188,6 @@ Warning: this option removes custom presets.</system:String>
   <system:String x:Key="pianoroll.menu.lyrics">Lyrics</system:String>
   <system:String x:Key="pianoroll.menu.lyrics.dashtoplus">Replace "-" with "+"</system:String>
   <system:String x:Key="pianoroll.menu.lyrics.edit">Edit Lyrics</system:String>
-  <system:String x:Key="pianoroll.menu.lyrics.hanzitopinyin">Hanzi to Pinyin</system:String>
```
Comment (stakira) [inline OpenUtau/Strings/Strings.axaml:191]: why remove this?

**Comment by stakira** [comment]
In case you don't know yet, the string files other than en-US are synced using `Misc/sync_strings.py` script.

File: OpenUtau/Strings/Strings.axaml (hunk 1 above)
Comment (maiko3tattun) [reply OpenUtau/Strings/Strings.axaml:191]: "menu.notes.hanzitopinyin" is used instead of "menu.lyrics.hanzitopinyin"

Comment (stakira) [reply OpenUtau/Strings/Strings.axaml:191]: great. thanks

**Pull request merged**